python/pipeline/utils/galvo_corrections.py

```diff
@@ -313,4 +315,277 @@ def correct_motion(scan, x_shifts, y_shifts, in_place=True):
                                     output=reshaped_scan[:, :, i])
 
     scan = np.reshape(reshaped_scan, original_shape)
-    return scan
\ No newline at end of file
+    return scan
+
+def _get_field_size_info(key):
+    """
+    Small utility function to fetch field height/width in pixels.
+    Required to make template functions below pipeline agnostic.
+    
+    Returns:
+        tuple: (height, width, height_microns_per_pixel, width_microns_per_pixel)
+    """
+    # Define virtual modules here to prevent circular imports
+    reso = dj.create_virtual_module("reso", "pipeline_reso")
+    meso = dj.create_virtual_module("meso", "pipeline_meso")
+
+    # Fetch relevant field size info
```

Comment:
technically these if statements will fail if the key is somehow present in both pipelines.  Worth creating a pipeline exception to error out if key is in both?